.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
-.idea
\ No newline at end of file
+.idea
+/vendor
```

Comment:
I'm adding the vendor file to the `.gitignore`. We should also decide what we want to do with our `composer.lock` files across all our projects. In most projects I've worked on, companies usually commit their `composer.lock` files so that all developers are on the same versions across all development computers. If we don't want to do that (so that we're always pulling something when we deploy, for example), then we should make the decision to add `composer.lock` to the `.gitignore`. What are your thoughts @daisy1754 ?

Reply:
I agree. I think we should add composer.lock to git